app/models/group.rb

```diff
@@ -40,7 +40,7 @@ class Group < Principal
   alias_attribute(:groupname, :lastname)
   validates_presence_of :groupname
   validate :uniqueness_of_groupname
-  validates_length_of :groupname, maximum: 30
+  validates_length_of :groupname, maximum: 256
```

Comment:
@oliverguenther Don't forget to also validate the length of `lastname` for `User`. Currently that still validates for a maximum length of 30.

Reply:
This was intentional for now as the requirement for user names haven't changed. However I'd also agree with increasing the limits there, as it's easy to run into 30 chars

Reply:
@wielinde are you willing to merge this without the changes to user? Do we also want to increase the limits in user names themselves? (/cc @ulferts)

Reply:
I'm fine with removing the restrictions on users as well

Reply:
@oliverguenther Actually, I thought lifting the limit was a requirement coming from some LDAP or whatever.... So I guessed that this could become an issue for normal users, too.

From a pure OpenProject perspective I find such long names completely useless. We will never ever be able to show such long names in a work package table as assignee for instance. It does not make sense from a UX perspective.

Reply:
The issue was exposed in the group synchronization, where we try to create OP groups with the same group name as the LDAP group. These LDAP groups tend to often have longer names and I think it's fine for us to just truncate them during displaying rather than truncating the name itself.

I will extend this for users